frontend/routes/package/score.tsx

```diff
@@ -164,6 +164,14 @@ export default function Score(
             This package is compatible with more than one runtime, and is marked
             as such in the package settings.
           </ScoreItem>
+          <ScoreItem
+            value={data.score.hasProvenance}
+            scoreValue={1}
+            title="Has provenance"
+          >
+            This package is published from a verifiable CI/CD workflow, and has
+            a public transparency log.
```

Comment:
Add links to docs

Reply:
Added a link, and updated the max down to 17. Let's see if this is widely adoptable, and once it is, we can tweak the max score :)